### ISSUE TITLE
'brew bump-cask-pr' does not work for SHA-only changes

### ISSUE THREAD

**Issue by core-code** [Homebrew/homebrew-cask#93013]
```
brew bump-cask-pr --force --message 'Automatically created and submitted by muUpdateStaticHomebrewCask' --no-browse --sha256 'de66546630615deb1bae58b54a3fc5f6bb29ec674c770c721b3245a29fad00cf' --version '5.3.0' securityspy

Error: You need to bump this cask manually since the new version
and old version are both 5.3.0.

```

this may be a design-desision but there are MANY checksum-only changes. they can be sent fine with 'cask-repair' but not with 'brew bump-cask-pr'.  while i agree this should probably be disabled by default to prevent faulty PRs, it should work with the '--force' flag

**Comment by miccal** [comment]
This is a design decision carried over from `brew bump-formula-pr`, but perhaps it could be changed for `brew bump-cask-pr`.

Ping @SeekingMeaning.

**Comment by SeekingMeaning** [comment]
> but there are MANY checksum-only changes

Perhaps these should be allowed by default, without needing `--force`

cc @vitorgalvao

**Comment by vitorgalvao** [comment]
I agree this should be doable. I have no strong preference if it should be behind `--force` or not; but it probably shouldn’t.

**Comment by core-code** [comment]
also
1.) it will not send downgrades even with the --force option. downgrades are quite common for casks

2.) it misdetects what a downgrade is. it thinks this upgrade is a downgrade:
```

brew bump-cask-pr --force --message 'Automatically created and submitted by muUpdateStaticHomebrewCask' --no-browse --sha256 '35693e38179cda5beabf8f4eff7bc9cc0c423086c248598734b74b393abde67d' --version '1.3.2,74' sensei

Error: You need to bump this cask manually since changing the
version from 1.3,71 to 1.3.2,74 would be a downgrade.

```

**Comment by core-code** [comment]
another valid update that is impossble to make because it is considered as a 'downgrade'

`brew bump-cask-pr --force  --no-browse --sha256 'f7067016df7488b6676d7dff16282796278a0b8e47d5e908b0a65b9609f06e6d' --version '1.4.1,135' touchswitcher;
`

**Comment by SeekingMeaning** [comment]
This should be fixed with `brew` version 2.6.2

**Comment by core-code** [comment]
awesome, thanks!
